core/electron-manager/src/ElectronPreload.ts

```diff
@@ -45,5 +40,5 @@ const frontendApi: IModelElectronApi = {
   },
 };
 
-// this adds the frontendApi object under the name `window.imodeljs_api` in the frontend Electron process.
-contextBridge.exposeInMainWorld("imodeljs_api", frontendApi);
+// this adds the frontendApi object under the name `window.itwinjs` in the frontend Electron process.

```

Comment:
@kabentley do we want to change this to itwinjs?  We'll have a mix of imodeljs and itwinjs in the APIs.

Reply:
yes, this is an example of something that should be iTwin as we evolve our apis. It's completely internal though, so it doesn't really count as part of the "api".